api/app/signals/apps/signals/factories/signal_user.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth import get_user_model
+from factory import DjangoModelFactory, Sequence, SubFactory, post_generation
+
+from signals.apps.users.models import SignalUser
```

Comment:
I believe the `SignalUser` model is in the wrong app. It should live in the `signals` app instead of the `users` app.
Or if we want it to be in the `users` app, then also all code beloning to this should be in the `users` app. For in case these factory classes.

Reply:
Agree, lets move it to signals app.